bin/aws-efs-creator.ts

```diff
@@ -30,16 +32,19 @@ appAspects.add(new ApplyTags({
   owner: owner,
 }));
 
+// security check
+appAspects.add(new AwsSolutionsChecks());
+
 const stackProps: AwsEfsCreatorStackProps = {
   resourcePrefix: `${appName}-${deployEnvironment}`,
   env: {
-      region: cdkRegion,
-      account,
+    region: cdkRegion,
+    account,
   },
   deployRegion: cdkRegion,
   deployEnvironment,
   appName,
-  vpcId: process.env.VPC_ID!,
+  vpcId,
 };
 new AwsEfsCreatorStack(app, `AwsEfsCreatorStack`, {
```

Comment:
**Suggestion:** Use a consistent naming convention for the `AwsEfsCreatorStack` identifier to improve code readability.
> 🌗  **Medium Priority** 
 > [category: maintainability, importance: 6]
```suggestion
new AwsEfsCreatorStack(app, `AwsEfsCreatorStack-${deployEnvironment}`, {
```Reason: Using a consistent naming convention that includes the deployment environment enhances readability and maintainability of the stack identifiers.